codalab/bundles/run_bundle.py

```diff
@@ -69,7 +69,9 @@ class RunBundle(DerivedBundle):
     METADATA_SPECS.append(MetadataSpec('exitcode', int, 'Exitcode of the process.', generated=True))
     METADATA_SPECS.append(MetadataSpec('job_handle', str, 'Identifies the job handle (internal).', generated=True, hide_when_anonymous=True))
     METADATA_SPECS.append(MetadataSpec('remote', str, 'Where this job is/was run (internal).', generated=True, hide_when_anonymous=True))
-    METADATA_SPECS.append(MetadataSpec('remotes', list, 'Previous workers where this job was run (internal); multiple values indicate that the bundle was preempted and moved to a different worker.', generated=True, hide_when_anonymous=True))
+    METADATA_SPECS.append(MetadataSpec('remotes', list, 'Previous workers where this job was run (internal); multiple values indicate that the bundle was preempted and moved to a different worker.',
```

Comment:
Remote names aren't verified / stable names, so I'm worried about depending on them for anything serious.

Reply:
We don't actually depend on them -- this field is not used otherwise at all. I just thought it would be good to keep track of which remotes it ran on for debugging purposes (as opposed to not collecting this info at all). Ideally we would store a list of {preemption time, remote name} -- but how useful do you think this is? Maybe we don't need to keep it at all?

Reply:
Just going to store list of remote names for now. At the very least, even if the names aren't useful, we'll know how many times the bundle was preempted.